addons/web/static/src/legacy/scss/bootstrap_overridden.scss

```diff
@@ -158,6 +158,7 @@ $toast-padding-y: 0.5rem !default;
 $toast-font-size: $font-size-base !default;
 $toast-background-color: rgba($white, .7) !default;
 $toast-header-background-color: $toast-background-color !default;
+$toast-box-shadow:  $o-floating-box-shadow !default;
```

Comment:
This will not do nothing  since it's used by the `.toast` class that you didn't applied (why?).
If you use that class you'll be able to remove the `box-shadow` currently in SCSS... maybe the `width` rule too?
Could you check?